90zfsbootmenu/zfsbootmenu.sh

```diff
@@ -344,6 +344,11 @@ while true; do
       "alt-r")
         emergency_shell "alt-r invoked"
         ;;
+      "alt-w")
+        pool="${selected_be%%/*}"
+        set_rw_pool "${pool}"
+        CLEAR_SCREEN=1 key_wrapper "${pool}"
```

Comment:
Should this be
```
if set_rw_pool "${pool}"; then
  CLEAR_SCREEN=1 key_wrapper "${pool}"
fi
```
to mirror other uses in `zfsbootmenu-lib.sh`?

Reply:
blah, yeah. Fixed.